### PR TITLE
Rebrand Unstake banner

### LINKED ISSUE #486: Rebrand DAO Dashboard - Unstake banner
## Requirements

Update Unstake Banner from the Staking page. [Designs](https://www.figma.com/design/NG6QClnVrvPRZ5gJ3s0zen/%E2%9F%81--Api3-DAO---Branding-Update?node-id=8676-107954&node-type=frame&m=dev)

Image

### PR DESCRIPTION
**Pull request by peterjurco**
Closes #486 

### What does this change?
- Updates styles for the Unstake banner on the Staking page

image

### CODE REVIEW

**Review by Anboias — APPROVED** [review]
👌

File: src/pages/dashboard/unstake-banner/unstake-banner.module.scss [hunk 1]
```diff
@@ -1,44 +1,52 @@
...
-    & > *:not(:last-child) {
-      margin-bottom: $space-lg;
-    }
+  background: linear-gradient(180deg, #fafafa 0%, #f0f4ff 100%);
+  border-top: 1px solid $color-dark-blue-10;
+  border-bottom: 1px solid $color-dark-blue-10;
+  margin: 32px 0;
+  padding: 24px 16px;
+
+  @media (min-width: $md) {
```
Comment (Anboias) [inline src/pages/dashboard/unstake-banner/unstake-banner.module.scss:17]: Actually I had a comment, not sure why it wasn't recorded when I approved the PR.

I was suggesting this change, which looks better imo

```suggestion
  @media (min-width: $sm) {
```

Comment (peterjurco) [reply src/pages/dashboard/unstake-banner/unstake-banner.module.scss:17]: it's better indeed

File: src/components/button/button.tsx [hunk 2]
```diff
@@ -19,7 +19,8 @@ export interface Props extends BreakpointsProps {
     | 'text'
     | 'text-blue'
     | 'menu-link-secondary'
-    | 'link-blue';
+    | 'link-blue'
+    | 'link-gray';
```
Comment (mcoetzee) [inline src/components/button/button.tsx:23]: Hmm wouldn't it make more sense to call this the `text-gray` variant? What's the connection with links here?

Comment (peterjurco) [reply src/components/button/button.tsx:23]: It's just a button type that is used in Figma. They have also "Text gray" button type but that's not what used here.

image

**Review by mcoetzee — APPROVED** [review]

**Pull request merged**